src/System.ServiceModel.Syndication/src/System/ServiceModel/Channels/UriGenerator.cs

```diff
@@ -12,26 +12,9 @@ internal class UriGenerator
         private long _id;
         private readonly string _prefix;
 
-        public UriGenerator() : this("uuid")
+        public UriGenerator()
         {
-        }
```

Comment:
Removing the constructor overloads is breaking change. It would break existing app using one of them.

Reply:
@Lxiamail These constructors are internal.

Reply:
Sorry, I didn't notice that UriGenerator is internal class. However, personally, I still see the current cosntructor overloading is nice to keep for potential future usage.

Reply:
Not really sure I see the point in keeping dead code like that around?